tools/targets/LPC.py

```diff
@@ -46,7 +44,7 @@ def is_patched(bin_path):
         return (sum([unpack('<I', bin.read(4))[0] for _ in range(8)]) & 0xFFFFFFFF) == 0
 
 
-if __name__ == "__main__":
-    binary = sys.argv[1]
-    print("LPC Patch: %s" % os.path.split(binary)[1])
-    patch(binary)
+if __name__ == '__main__':
+    bin_path = "C:/Users/emimon01/releases/emilmont/build/test/LPC1768/ARM/MBED_A1/basic.bin"
```

Comment:
Looks wrong.

Reply:
Yes, this is how it was and we are not touching it anymore, please refresh

Reply:
@rwalton-arm it is indeed how it was and we are not touching it.